SparkPost/matchbox
# FE-987 Set checked prop in toggle

Give your PR a recognizable title. For example: **FE-123: Ticket Title** or **Resolve Issue #123: Description**.

### What Changed
- Correctly sets checked prop in toggle
- Adds defaultChecked prop

### How To Test or Verify
- `npm run start:storybook`
- verify tests are passing
- verify toggle is being checked by checked prop

### PR Checklist
